### ISSUE TITLE
ADO task does not fail on a11y issues in non-PR builds

### ISSUE THREAD

**Issue by pownkel** [microsoft/accessibility-insights-action#920]
The ADO build task doesn't fail on accessibility errors in our manually-triggered pipeline when `failOnAccessibilityErrors` is set to `true`. @karanbirsingh pointed out that the failOnAccessibilityError function only appears in the pull request comment creator, so it is expected to also not fail on errors in CI pipelines.

#### To reproduce:
1. Create an ADO build pipeline with the ADO extension task and a url with known accessi
2. set `failOnAccessibilityError: true` and `url: https://www.washington.edu/accesscomputing/AU/before.html` (or some other page with known a11y failures)
3. Trigger the build manually. The logs will show that failures were found, but the build will not break.

**Comment by dbjorge** [comment]
Coved by #942

**Comment by dbjorge** [comment]
Fix released in v1.1.0

**Comment by dbjorge** [comment]
Demo of the issue being fixed in a manually triggered pipeline against v1.1.0 (from the staging channel) at https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=28323&view=results (see case 7)
